src/main/java/in/projecteka/consentmanager/clients/model/LinkedCareContexts.java

```diff
@@ -89,4 +103,13 @@ public boolean hasCCReferences(String hipId, List<String> ccRefs) {
         }
         return result;
     }
+
+    public List<PatientLinkedCareContext> getCareContext(String hipId) {
+        return links.stream().filter(hipLink -> hipLink.hipId.equals(hipId))
+                .flatMap(hipLink -> hipLink.careContexts.stream().map(careContext ->
+                        PatientLinkedCareContext.builder()
+                                .careContextRefNo(careContext.careContextReference)
+                                .patientRefNo(hipLink.patientReference)
+                                .build())).collect(Collectors.toList());
```

Comment:
you can use collectList

Reply:
Sure.

Reply:
collectList is a Flux Method, we are not using Flux in this method 'getCareContext'.

Reply:
pl